docs/content/configuration/creating_volumereplication_pbr.md

```diff
@@ -0,0 +1,47 @@
+# Creating a VolumeReplication with policy-based replication
+
+Create a VolumeReplication YAML file to replicate a specific PersistentVolumeClaim (PVC).
+
+VolumeReplicationClass needs to be present before a VolumeReplication can be created. For more information, see [Creating a VolumeReplicationClass](creating_volumereplicationclass.md).
+
+**Note:** For information and parameter definitions that are not related to topology awareness, be sure to see the information provided in [Creating a VolumeReplication](creating_volumereplication.md), in addition to the current section.
+
+**Note:** Use the `VolumeGroup` value for `spec.dataSource.kind`.
+
+1.  Replicate a specific PersistentVolumeClaim (PVC) using the `demo-volumereplication.yaml`.
+
+    For more information about PVC configuration, see [Creating a PersistentVolumeClaim (PVC)](creating_pvc.md).
+
+    ```
+    apiVersion: replication.storage.openshift.io/v1alpha1
+    kind: VolumeReplication
+    metadata:
+      name: demo-volumereplication
+      namespace: default
+    spec:
+      volumeReplicationClass: demo-volumereplicationclass
+      replicationState: primary
+      dataSource:
+        kind: VolumeGroup
+        name: demo-pvc-file-system  # Ensure that this is in the same namespace as VolumeReplication.
```

Comment:
this should be the name of the volume group

Reply:
@ELENAGER